modules/oauth-configuring-token-inactivity-timeout.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-internal-oauth.adoc
+
+[id="oauth-token-inactivity-timeout_{context}"]
+= Configuring token inactivity timeout for the internal OAuth server
+
+You can configure OAuth tokens to expire after a certain amount of time that they have been inactive. By default, there is no token inactivity timeout set.
+
+[NOTE]
+====
+If token inactivity timeout is also configured in your OAuth client, that value overrides the timeout set in the internal OAuth server configuration.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured a password-based identity provider (IDP), such as HTPasswd or LDAP.
+
+.Procedure
+
+. Update the OAuth configuration to set a token inactivity timeout.
+
+.. Edit the OAuth configuration:
++
+[source,terminal]
+----
+$ oc edit oauth cluster
+----
++
+Add the `spec.tokenConfig.accessTokenInactivityTimeout` field and set your timeout value:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+...
+spec:
+  tokenConfig:
+    accessTokenInactivityTimeout: 400s <1>
+----
+<1> Set a value with the appropriate units, for example `400s` for 400 seconds, or `30m` for 30 minutes. The minimum allowed timeout value is `300s`.
+
+.. Save the file to apply the changes.
+
+. Wait for the OAuth server Pods to restart:
++
+[source,terminal]
+----
+$ oc get clusteroperators authentication
+----
++
+Do not continue to the next step until `PROGRESSING` is listed as `False`, as shown in the following output:
++
+.Example output
+[source,terminal]
+----
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+authentication   4.6.0     True        False         False      145m
+----
+
+. Wait for a new revision of the Kubernetes API server Pods to roll out. This will take several minutes.
++
+// TODO: Note that if you do this quickly enough, it will say False before it turns to True, then goes to False after ~10 minutes.
++
+[source,terminal]
+----
+$ oc get clusteroperators kube-apiserver
+----
++
+Do not continue to the next step until `PROGRESSING` is listed as `False`, as shown in the following output:
++
+.Example output
+[source,terminal]
+----
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+kube-apiserver   4.6.0     True        False         False      145m
+----
++
+If `PROGRESSING` is showing `True`, wait a few minutes and try again.
+
+. Verify that the token inactivity timeout was successfully set.
+
+.. Log in to the cluster with an identity from your password-based IDP.
+
+.. Execute a command and verify that it was successful.
+
+.. Wait longer than the configured timeout without using the identity. In this procedure's example, wait longer than five minutes.
```

Comment:
Perhaps "wait longer than 300 seconds"? But in the sample spec, you show `400s`. 

Also, my understanding with ISG is that you would write it as "5 minutes" (use a numeral with numbers combined with a unit of measurement, from p. 157).